### PR TITLE
Add test 'Page for terms and conditions'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.7.x
| Description?      | Add test 'Page for terms and conditions'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | no
| Possible impacts? | TEST_PATH="functional/BO/13_shopParameters/02_orderSettings/01_orderSettings/general/06_termsOfService.js" URL_FO=yourShopURL npm run specific-test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/22543)

### CODE REVIEW

File: tests/UI/pages/FO/checkout/index.js [hunk 1]
```diff
@@ -223,6 +225,11 @@ class Checkout extends FOBasePage {
     return this.elementVisible(page, this.conditionToApproveCheckbox, 1000);
   }
 
+  async getTermsOfServicePageTitle(page) {
```
Comment (boubkerbribri) [inline tests/UI/pages/FO/checkout/index.js:233]: Please add function description

File: tests/UI/pages/FO/checkout/index.js [hunk 2]
```diff
@@ -223,6 +225,11 @@ class Checkout extends FOBasePage {
...
+    await page.click(this.termsOfServiceLink);
```
Comment (boubkerbribri) [inline tests/UI/pages/FO/checkout/index.js:234]: If it's a navigation link, please use this instead

```suggestion
    await this.clickAndWaitForNavigation(page, this.termsOfServiceLink);
```

Comment (nesrineabdmouleh) [reply tests/UI/pages/FO/checkout/index.js:234]: No, it's not a navigation link, when we click a modal is displayed.

File: tests/UI/pages/FO/checkout/index.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/FO/checkout/index.js:233]: Done thx :+1:

File: tests/UI/pages/FO/checkout/index.js (hunk 2 above)
Comment (boubkerbribri) [reply tests/UI/pages/FO/checkout/index.js:234]: ok thanks

**Review by boubkerbribri — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh & @boubkerbribri

**Pull request merged**